pkg/kubelet/volumes.go

```diff
@@ -78,7 +78,7 @@ func (vh *volumeHost) NewWrapperCleaner(spec *volume.Spec, podUID types.UID) (vo
 	return c, nil
 }
 
-func (kl *Kubelet) newVolumeBuilderFromPlugins(spec *volume.Spec, podRef *api.ObjectReference, opts volume.VolumeOptions) (volume.Builder, error) {
+func (kl *Kubelet) newVolumeBuilderFromPlugins(spec *volume.Spec, podRef *api.ObjectReference, opts volume.VolumeOptions, secretClient volume.SecretClient) (volume.Builder, error) {
```

Comment:
It made testing a snap, which I think speaks well to the value of having narrow interfaces for the task at hand.